spec/fixtures/description-license/DESCRIPTION

```diff
@@ -0,0 +1,2 @@
+Package: xyz
+License: MIT + file LICENSE
```

Comment:
If the author is saying this is licensed under the MIT license + whatever arbitrary terms I put in the license file, why is it accurate to say the project is licensed under the MIT license? (e.g., if the license file says "and you must give me 100 puppies if you use this software")